### PR TITLE
en, zh: update aws deploy doc

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by handlerww — APPROVED** [review]
LGTM

File: zh/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -75,9 +122,14 @@ nodeGroups:
...
-> **注意：**
+该命令需要等待 EKS 集群创建完成，以及节点组创建完成并加入进去，耗时约 5~10 分钟。可参考 [eksctl 文档](https://eksctl.io/usage/creating-and-managing-clusters/#using-config-files)了解更多集群配置选项。
+
+> **警告：**
+>
+> 如果使用了 Regional ASG (Auto Scaling group)：
 >
-> 该命令需要等待 EKS 集群创建完成，以及节点组创建完成并加入进去，耗时约 5~10 分钟。可参考 [eksctl 文档](https://eksctl.io/usage/creating-and-managing-clusters/#using-config-files)了解更多集群配置选项。
+> * 参考 AWS [文档](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-instance-termination.html#instance-protection-instance) 对已经启动的 EC2 打开实例缩减保护，ASG 自身的实例缩减保护不需要打开。
+> * 参考 AWS [文档](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-instance-termination.html#custom-termination-policy)设置 ASG 终止策略为 `NewestInstance`。
```
Comment (ran-huang) [inline zh/deploy-on-aws-eks.md:132]:
```suggestion
> * 参考 [AWS 文档](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-instance-termination.html#instance-protection-instance) 对已经启动的 EC2 打开实例缩减保护，ASG 自身的实例缩减保护不需要打开。
> * 参考 [AWS 文档](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-instance-termination.html#custom-termination-policy)设置 ASG 终止策略为 `NewestInstance`。
```

File: zh/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -413,6 +496,8 @@ AWS 部分实例类型提供额外的 [NVMe SSD 本地存储卷](https://docs.aw
 > 运行中的 TiDB 集群不能动态更换 storage class，可创建一个新的 TiDB 集群测试。
 >
 > 由于 EKS 升级过程中节点重建，[本地盘数据会丢失](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-lifetime)。由于 EKS 升级或其他原因造成的节点重建，会导致需要迁移 TiKV 数据，如果无法接受这一点，则不建议在生产环境中使用本地盘。
+>
+> 由于节点重建会导致本地存储数据丢失，请参考 AWS [文档](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-suspend-resume-processes.html)停止 TiKV 节点组的 `ReplaceUnhealthy` 功能。
```
Comment (ran-huang) [inline zh/deploy-on-aws-eks.md:500]:
```suggestion
> 由于节点重建会导致本地存储数据丢失，请参考 [AWS 文档](https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-suspend-resume-processes.html)停止 TiKV 节点组的 `ReplaceUnhealthy` 功能。
```

**Comment by DanielZhangQD** [comment]
@ran-huang PTAL again.

File: en/deploy-on-aws-eks.md [hunk 3]
```diff
@@ -36,30 +38,85 @@ apiVersion: eksctl.io/v1alpha5
...
 nodeGroups:
   - name: admin
     desiredCapacity: 1
+    privateNetworking: true
     labels:
       dedicated: admin
 
-  - name: tidb
-    desiredCapacity: 2
+  - name: tidb-1a
```
Comment (tennix) [inline en/deploy-on-aws-eks.md:50]: TiDB doesn't have to be deployed 3 replicas.

File: en/deploy-on-aws-eks.md [hunk 4]
```diff
@@ -262,12 +324,12 @@ This section describes how to scale out the EKS node group and TiDB components.
...
 ### Scale out EKS node group
 
-The following example shows how to scale out the `tikv` group of the `${clusterName}` cluster to 4 nodes:
+The following example shows how to scale out the `tikv-1a` group of the `${clusterName}` cluster to 2 nodes:
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-eksctl scale nodegroup --cluster ${clusterName} --name tikv --nodes 4 --nodes-min 4 --nodes-max 4
+eksctl scale nodegroup --cluster ${clusterName} --name tikv-1a --nodes 2 --nodes-min 2 --nodes-max 2
```
Comment (tennix) [inline en/deploy-on-aws-eks.md:332]: Scaling out needs to be evenly in 3 AZs for TiKV.

**Review by ran-huang — COMMENTED** [review]
LGTM

**Review by ran-huang — COMMENTED** [review]
LGTM

File: en/deploy-on-aws-eks.md (hunk 3 above)
Comment (DanielZhangQD) [reply en/deploy-on-aws-eks.md:50]: Updated

File: en/deploy-on-aws-eks.md (hunk 4 above)
Comment (DanielZhangQD) [reply en/deploy-on-aws-eks.md:334]: Updated.

**Comment by DanielZhangQD** [comment]
@tennix comments addressed, PTAL again.

**Review by tennix — APPROVED** [review]
LGTM

File: en/deploy-on-aws-eks.md [hunk 5]
```diff
@@ -262,12 +326,14 @@ This section describes how to scale out the EKS node group and TiDB components.
 
 ### Scale out EKS node group
 
-The following example shows how to scale out the `tikv` group of the `${clusterName}` cluster to 4 nodes:
+For TiKV, we need to scale out evenly among the different availability zones, and the following example shows how to scale out the `tikv-1a`, `tikv-1c`, and `tikv-1d` groups of the `${clusterName}` cluster to 2 nodes:
```
Comment (ran-huang) [inline en/deploy-on-aws-eks.md:329]:
```suggestion
When scaling out TiKV, the node groups must be scaled out evenly among the different availability zones. The following example shows how to scale out the `tikv-1a`, `tikv-1c`, and `tikv-1d` groups of the `${clusterName}` cluster to 2 nodes:
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: zh/deploy-on-aws-eks.md [hunk 6]
```diff
@@ -37,36 +39,93 @@ apiVersion: eksctl.io/v1alpha5
...
+    desiredCapacity: 1
+    privateNetworking: true
+    availabilityZones: ["ap-northeast-1c"]
     labels:
       dedicated: tikv
     taints:
       dedicated: tikv:NoSchedule
 ```
 
+默认我们只需要两个 TiDB 节点，因此我们设置 `tidb-1d` 节点组的 `desiredCapacity` 为 `0`，后面如果需要我们可以随时扩容这个节点组。
```
Comment (DanielZhangQD) [inline zh/deploy-on-aws-eks.md:127]:
```suggestion
默认只需要两个 TiDB 节点，因此可以设置 `tidb-1d` 节点组的 `desiredCapacity` 为 `0`，后面如果需要可以随时扩容这个节点组。
```

File: zh/deploy-on-aws-eks.md [hunk 7]
```diff
@@ -261,12 +325,14 @@ basic-grafana   LoadBalancer   10.100.199.42   a806cfe84c12a4831aa3313e792e3eed-
 
 ### 扩容 EKS 节点组
 
-以下是将集群 `${clusterName}` 的 `tikv` 节点组扩容到 4 节点的示例：
+TiKV 扩容需要保证在各可用区均匀扩容，以下是将集群 `${clusterName}` 的 `tikv-1a`、`tikv-1c`、`tikv-1d` 节点组扩容到 2 节点的示例：
```
Comment (DanielZhangQD) [inline zh/deploy-on-aws-eks.md:328]:
```suggestion
TiKV 扩容需要保证在各可用区均匀扩容。以下是将集群 `${clusterName}` 的 `tikv-1a`、`tikv-1c`、`tikv-1d` 节点组扩容到 2 节点的示例：
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**